mm2src/coins/lp_coins.rs

```diff
@@ -3031,14 +3031,28 @@ pub trait MmCoin:
     /// Loop collecting coin transaction history and saving it to local DB
     fn process_history_loop(&self, ctx: MmArc) -> Box<dyn Future<Item = (), Error = ()> + Send>;
 
+    #[cfg(not(target_arch = "wasm32"))]
+    fn account_db_id(&self) -> Result<Option<String>, String> { Ok(None) }
+
+    #[cfg(target_arch = "wasm32")]
+    fn account_db_id(&self) -> Result<Option<String>, String> {
+        Ok(Some(
+            try_s!(Public::from_slice(
+                try_s!(hex::decode(try_s!(self.get_public_key()))).as_slice()
+            ))
+            .address_hash()
+            .to_string(),
+        ))
+    }
```

Comment:
Wouldn't this cause breaking changes (ignoring existing db files of coins and trying to connect/create new ones)? Due to that concern I thought returning `None` by default would be the right approach here. So we can return different db paths only when it's needed.

Reply:
Agreed!